CMakeLists.txt

```diff
@@ -79,6 +79,19 @@ SET(CMAKE_BUILD_TYPE "${CMAKE_BUILD_TYPE}" CACHE STRING
     "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel Maintainer."
     FORCE)
 
+# Explicit request to use CCACHE
+if($ENV{USE_CCACHE})
```

Comment:
After having a look at both

https://docs.travis-ci.com/user/migrating-from-legacy/
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

it seems like we can actually benefit from the memory increase from using  `sudo: required` and increase the number of workers in all make tasks